api/RTCPeerConnection.json

```
@@ -2647,10 +2647,10 @@
                 "version_added": "66"
               },
               "safari": {
-                "version_added": "preview"
+                "version_added": "15.4"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "15.4"
               },
```

Rather than manually mirror, we can automatically mirror the data to improve contributor experience:

```suggestion
              "safari_ios": "mirror",
```

---

Oh, excellent! I didn't know about the `"mirror"` syntax, but that makes really good sense to me. I'll commit the suggestions here inline, but if you'd prefer me to redo and push as a fresh commit, let me know that too :)

---

Ah, looks like no dice on that. Stand by and I'll push a commit.